lti_lib/student_score.js

```diff
@@ -0,0 +1,24 @@
+const axios = require('axios');
+
+function grading_grade(grade, payload) {
+
```

Comment:
you need to check the scope to make sure the LMS wants to receive the grade.  This is where you check the `payload["https://purl.imsglobal.org/spec/lti-ags/claim/endpoint"].scope` array to see if it includes `'https://purl.imsglobal.org/spec/lti-ags/scope/score'`